src/api/entity/requests.rs

```diff
@@ -0,0 +1,16 @@
+use super::responses::CreateEntityResponse;
+use rustify_derive::Endpoint;
+use std::fmt::Debug;
+
```

Comment:
Please add some Comments above your function for example.
```suggestion
/// ## Create Entity
/// This endpoint creates an entity.
///
/// * Path: /identity/entity
/// * Method: POST
/// * Response: [CreateEntityResponse]
/// * Reference: https://www.vaultproject.io/api-docs/dentity/entity#create-an-entity
```